config/foreman-proxy-content.migrations/160120212700-foreman-proxy.rb

```diff
@@ -29,7 +29,7 @@ def mod(manifest, params)
 answers['foreman_proxy'] = {
   'custom_repo' => true,
   'http' => true,
-  'ssl_port' => '9090',
```

Comment:
Migrations are executed only once and then it's stored that it was executed. That means you have to create new migrations. There's a helper script ```bin/create-migration```